.github/workflows/ci.yaml

```diff
@@ -6,32 +6,20 @@ on:
 
 jobs:
   lint:
-    name: ${{ matrix.toxenv }}
+    name: Lint
```

Comment:
nit: keep this lowercase to match `toxenv` usage elsewhere?

Reply:
No problem. Changed in latest.